pkg/collector/corechecks/snmp/checkconfig/config.go

```diff
@@ -180,7 +181,8 @@ func (c *CheckConfig) RefreshWithProfile(profile string) error {
 
 // UpdateDeviceIDAndTags updates DeviceID and DeviceIDTags
 func (c *CheckConfig) UpdateDeviceIDAndTags() {
-	c.DeviceID, c.DeviceIDTags = buildDeviceID(c.getDeviceIDTags())
+	c.DeviceIDTags = util.SortUniqInPlace(c.getDeviceIDTags())
```

Comment:
Should you get rid of `buildDeviceID` as it's not used anymore?

Reply:
Yes, but, I just wanted to make the scope/codeChange of this PR as small as possible since it’s for an RC. Let me know if you prefer we remove it in this PR right away.